modules/web-element.mjs

```diff
@@ -324,22 +324,188 @@ class PaddingLeft extends WebElementFunction {
+        super("margin-bottom")
+    }
+    get(element){
+        var doc = element.ownerDocument;
+        var win = doc.defaultView || doc.parentWindow
+        var elem = doc.querySelector('.testElem')
+        var cssProp = win.getComputedStyle(elem).getPropertyValue('margin-bottom')
+        return cssProp
+    }
+}
+/**
+ * Function that extract margin-left of a DOM
+ */
+class CheckMarginLeft extends WebElementFunction{
+    constructor(){
+        super("margin-left")
+    }
+    get(element){
+        var doc = element.ownerDocument;
+        var win = doc.defaultView || doc.parentWindow
+        var elem = doc.querySelector('.testElem')
+        var cssProp = win.getComputedStyle(elem).getPropertyValue('margin-left')
+        return cssProp
+    }
+}
+/**
+ * Function that extract margin-right of a DOM
+ */
+class CheckMarginRight extends WebElementFunction{
+    constructor(){
+        super("margin-right")
+    }
+    get(element){
+        var doc = element.ownerDocument;
+        var win = doc.defaultView || doc.parentWindow
+        var elem = doc.querySelector('.testElem')
+        var cssProp = win.getComputedStyle(elem).getPropertyValue('margin-right')
+        return cssProp
+    }
+}
+/**
+ * Function that extract paddig-top of a DOM
+ */
+class CheckPaddingTop extends WebElementFunction{
+    constructor(){
+        super("padding-top")
+    }
+    get(element){
+        var doc = element.ownerDocument;
+        var win = doc.defaultView || doc.parentWindow;
+        var elem = doc.querySelector('.testElem')
+        var cssProp = win.getComputedStyle(elem).getPropertyValue('padding-top')
+        return cssProp
+    }
+}
+/**
+ * Function that extract paddig-bottom of a DOM
+ */
+class CheckPaddingBottom extends WebElementFunction{
+    constructor(){
+        super("padding-bottom")
     }
-    get(e){
-        var s = e.style.fontFamily
-        return s
+    get(element){
+        var doc = element.ownerDocument;
+        var win = doc.defaultView || doc.parentWindow;
+        var elem = doc.querySelector('.testElem')
+        var cssProp = win.getComputedStyle(elem).getPropertyValue('padding-bottom')
+        return cssProp
     }
 }
+/**
+ * Function that extract paddig-left of a DOM
+ */
+class CheckPaddingLeft extends WebElementFunction{
+    constructor(){
+        super("padding-left")
+    }
+    get(element){
+        var doc = element.ownerDocument;
+        var win = doc.defaultView || doc.parentWindow;
+        var elem = doc.querySelector('.testElem')
+        var cssProp = win.getComputedStyle(elem).getPropertyValue('padding-left')
+        return cssProp
+    }
+}
+/**
+ * Function that extract paddig-right of a DOM
+ */
+class CheckPaddingRight extends WebElementFunction{
+    constructor(){
+        super("padding-right")
+    }
+    get(element){
+        var doc = element.ownerDocument;
+        var win = doc.defaultView || doc.parentWindow;
+        var elem = doc.querySelector('.testElem')
```

Comment:
The element to test is already available as it is passed as an argument (`element`).
Therefore, line 503 should be removed, and line 504 should use the existing `element` instead of `elem`.

The same applies for all other functions in this PR as well.